src/js/nav/globalNav.js

```diff
@@ -10,24 +13,24 @@ export let getNavFromConfig = async (masterConfig) => {
     }, {});
 };
 
+const isCurrVisible = (permissions) => Promise.all(
+    flatMap([permissions], ({ method } = {}) => (
+        // (null, undefined, true) !== false
```

Comment:
leftover comment?

Reply:
Not really, this is to show what is actually evalueted here because the boolean logic is slightly more complex than it could be for new commers.